### ISSUE TITLE
Slash in wrong direction on bootstrap-vcpkg.bat

### ISSUE THREAD

**Issue by SamuelMarks** [microsoft/vcpkg#17255]
Super minor but yeah `E:\my_dir\vcpkg/vcpkg.exe` instead of `E:\my_dir\vcpkg\vcpkg.exe` in command prompt at c2af80b1bfadea426857a443c7c38a4ec3b0f46c:
```
E:\my_dir\vcpkg>bootstrap-vcpkg.bat
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2021-02-24-d67989bce1043b98092ac45996a8230a059a2d7e/vcpkg.exe -> E:\my_dir\vcpkg/vcpkg.exe
Done.

Telemetry
---------
vcpkg collects usage data in order to help us improve your experience.
The data collected by Microsoft is anonymous.
You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disableMetrics,
passing --disable-metrics to vcpkg on the command line,
or by setting the VCPKG_DISABLE_METRICS environment variable.

Read more about vcpkg telemetry at docs/about/privacy.md
```

**Comment by PhoebeHui** [comment]
@SamuelMarks, thanks for reporting this issue! I have submitted a PR to fix it.

**Comment by SamuelMarks** [comment]
Great, this will be closed by #17248
